icon.schema.json

```diff
@@ -51,6 +70,12 @@
     "deprecated": {
       "type": "boolean",
       "default": false
+    },
+    "deprecationReason": {
+      "type": "string"
+    },
```

Comment:
We can also enforce the relation ship here.

Reply:
I've added a `dependentRequired` clause on both instances.